docs/guidelines/content-guidelines/05-tabs.md

```diff
@@ -0,0 +1,237 @@
+
+## Documentation toggle
+
+The toggle renders the text in Github documents in the form of expandable sections and shows it as tabs on the `http://kyma-project.io` website.
+
+Content from documentation toggles is displayed as tabs on `http://kyma-project.io` using a dedicated script.
+
+### Writing rules
+
+Follow these rules when inserting toggles in documents:
+
+* Toggles belonging to one set must start with the `<div tabs name="{toggle-name}">` tag and end with the `</div>` tag, where **name** is a distinctive ID of a specific toggle used for linking. It can be any name describing the content of the documentation toggle.
+
+See the example:
+
+```markdown
+<div tabs name="installation">
+{toggle content}
+</div>
+```
+
+* A single toggle must start with the `<details>` tag and end with the `</details>` tag.
+* Insert the title of the toggle between `<summary>` and `</summary>` tags. Every part of the title must start from a new line.
+
+See the example:
+
+``` markdown
+<summary>
+From release
+</summary>
+```
+
+* Leave a blank line after the `</summary>` tag. Only this way the content can render.
+* Add content using Markdown syntax.
+* Use this indentation when you add the toggle as a regular document section:
+
+  - No spaces for the div element
+  - Two spaces for other elements
+
+* Use this indentation when you add a toggle under a list:
+
+  - Four spaces for the div element
+  - Six spaces for other elements
+
+* When you add two or more toggle sets that belong to the same group, you must classify them properly. This way each time you switch to one of the toggles in the set, it opens automatically in all other sets from the same group within one document and across all documents that use the group. To create a toggle group:
+
+  - All sets of toggles from the given group must contain the `group` attribute with a distinctive ID of this group. For example, add the `cluster-installation` group to categorize all toggle sets that refer to Kyma installation on a cluster.
+  - Every `<summary>` tag in the set must also contain the `label` attribute. For example, add the `GKE` label to all toggles that describe steps for Kyma installation on a GKE cluster in the `cluster-installation` group.
+
+    > **NOTE:** The label does not have to be the same as the summary. If the label is a combination of two or more words, separate them with the dash (`-`). For example, write `<summary label="install-on-gke">`.
+
+  See the example:
+
+  ``` markdown
+  <div tabs name="prerequisites" group="cluster-installation">
+    <details>
+    <summary label="gke">
+    GKE
+    </summary>
+    ...
+    </details>
+    <details>
+    <summary label="aks">
+    AKS
+    </summary>
+    ...
+    </details>
+    <details>
+    <summary label="gardener">
+    Gardener
+    </summary>
+    ...
+    </details>
+  </div>
+  ```
+
+### Examples
```

Comment:
Oh, now I realize the example above probably only belongs to the one bullet point :D